playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
```

Comment:
Tags are used in various places throughout our playbooks so for consistency in entry point playbook configuration I added them here.  Usage of tags works in some playbooks and others it doesn't so ultimately we should either fix tag usage or remove them altogether.  To answer your specific questions, using `tags: always` allows someone to run the playbook specifying a tag or tags to run a certain subsection of the playbooks, but will always run the plays/tasks with the `always` tag.  Since these playbooks are part of initialization, we always want them to run.